pkg/render/node.go

```diff
@@ -215,7 +216,10 @@ func (c *nodeComponent) Objects() ([]client.Object, []client.Object) {
 		objs = append(objs, c.nodePodSecurityPolicy())
 	}
 
-	objs = append(objs, c.nodeDaemonset(cniConfig))
+	// Note: this will override the values from nodeResources() (if it was defined for calico-node).
+	overrides := c.cfg.Installation.CalicoNodeDaemonSet
+	ds := rcomp.ApplyDaemonSetOverrides(c.nodeDaemonset(cniConfig), overrides)
```

Comment:
I think it would be nicer to put this within the `c.nodeDaemonset` function call, so the complete generation of the daemonset is handled within that function.

Reply:
Fixed